docs/api-docs/cmds.md

```diff
@@ -261,16 +75,20 @@ return loop(
 );
 ```
 
-
 ### `Cmd.run(func, options)`
```

Comment:
I just tried to read through the Cmd documentation and I feel like it would be easier to read if we moved the notes for each one above the parameters and the simulation instructions. Sorry I know you didn't do that either. It was just weird having to read a bunch of usage details before seeing an explanation of what it does.

Reply:
or do you think it would read better to have a short description above and the detailed notes below (where they are now)

Reply:
yeah, no, I agree with your first idea. Let me give that a shot first and see how it reads..